aruco_detect/src/aruco_detect.cpp

```diff
@@ -360,7 +360,7 @@ void FiducialsNode::imageCallback(const sensor_msgs::ImageConstPtr & msg) {
                          tvecs[i][0], tvecs[i][1], tvecs[i][2],
                          rvecs[i][0], rvecs[i][1], rvecs[i][2]);
 
-                if (ignoreIds.find(ids[i]) != ignoreIds.end()) {
+                if (std::count(ignoreIds.begin(), ignoreIds.end(), ids[i]) != 0) {
                     ROS_INFO("Ignoring id %d", ids[i]);
                     continue;
                 }
```

Comment:
Wouldn't this be slower, since than map.find(), since it traverses the vector?

Reply:
Not a big deal unless the number of items is very large, but I don't see the benefit of changing a map to a vector.

Reply:
The reason for not using map is because the second 'value' element is useless, and only adds noise to the code, since we only care if it is in the list to ignore or not.

I debated using `set` (which is like map but with only a key) and `set.count()` but since the typical application of this is for very few fiducials (<10) vector beats both map and set in terms of speed of counting due to a lower memory footprint and contiguous accesses. The performance of this is not a big deal, but either way: for small numbers of ignored ids, vector should be the fastest.